### ISSUE TITLE
ssh failure between xenial nodes

### ISSUE THREAD

**Issue by raviram203** [HSBawa/icp-ce-on-linux-containers#2]
hi, Hari. 
All lxc nodes reach running state and have two network interfaces, but ssh setup seems awry. During one run I noticed ssh failure messages in the install log file in dev-boot. First time, I had not installed kubectl & IBM Cloud CLI. The second & third times (lxd check all OK), the log file was empty. All three times the `terraform apply` command finished in 2 minutes. dev-boot had no running docker containers, but had the `ibmcom/icp-inception   2.1.0.2` docker image and extracted `/opt/icp-2.1.0.2-ce` installer. There was no `cfc` folder. I am trying to build ICP ce 2102 and pulled a fresh copy from github. Am using the same machine which worked built ICP ce 2101. Have you seen this before?
Thanks, Ravi

**Comment by HSBawa** [comment]
Hi Ravi, Today, I am also facing issue for the reason Linux Container nodes are restarting arbitrarily during install (making it unreachable). Not sure what is causing this, but, I will take a look and keep posted.

**Comment by HSBawa** [comment]
@raviram203 
Following is my observation and  workaround: 

* For some reason, nodes restart is happening only when ICP is installed as part of terraform apply. This issue didn't pop up when I did ICP install manually from Boot Node (do not run terraform destroy or terra clean script):
  * lxc exec dev-boot bash
  * cd /opt/icp-2.1.0.2-ce/bin
  * ./uninstall.sh  /opt/icp-2.1.0.2-ce/cluster ibmcom/icp-inception 2.1.0.2  /opt/icp-2.1.0.2-ce/log/    
  * ./install.sh  /opt/icp-2.1.0.2-ce/cluster ibmcom/icp-inception 2.1.0.2  /opt/icp-2.1.0.2-ce/log/    

 See if above workaround applies in your case. I will also take a look into node restart issue.
![image](https://user-images.githubusercontent.com/19266024/38331039-270619d6-3820-11e8-8944-6af7ac5e3325.png)

**Comment by raviram203** [comment]
Oops. I did not see your advice until now. I winged it and managed to install ICP 2102. Just confirmed the version in the UI console. I gutted the lxc nodes with the manual destroy script, cloned the git repo, and applied the terraform plan (2 minutes). ICP was not running. I ran the install command from dev-boot after I saw that the ICP docker image was on dev-boot. Took 41 minutes and worked, even though I had not run the uninstall. I did not verify if ssh from boot to other nodes was working. Just ran the install script while I was working on the work laptop. Thanks for fixing the problem.
